apps/ui/src/lib/strapi.ts

```diff
@@ -40,10 +45,11 @@ export default class Strapi {
     const { url, headers } = await this.prepareRequest(
       path,
       params,
-      options?.strapiJWT
+      options?.strapiJWT,
+      options?.omitAuthorization
     )
     const response = await fetch(url, {
-      cache: "no-cache", // `next: { revalidate: 60 }`
+      next: { revalidate: env.NEXT_PUBLIC_REVALIDATE }, // cache: "no-cache" // for dynamic apps
```

Comment:
The only discussion I'd like to open is whether we want all the requests to have the same  revalidate period. This might not be true for all page types (e.g. blogs would revalidate more often than a contact page). Perhaps we could choose a middle ground and atleast add an override option to this method's parameters.

Reply:
There is always option to set specific period for request passing `requestInit`:

```ts
   const response = await fetch(url, {
      next: { revalidate: env.NEXT_PUBLIC_REVALIDATE }, // cache: "no-cache" // for dynamic apps
      ...requestInit,
      headers: { ...requestInit?.headers, ...headers },
    })
```